AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -31,6 +39,7 @@ class CaptureImageViewController: UIViewController, UIImagePickerControllerDeleg
         self.dismissViewControllerAnimated(true, completion: nil)
         self.image = info[UIImagePickerControllerOriginalImage] as! UIImage
         imageView.image = self.image
+        self.data = self.compressFile() as? NSData
```

Comment:
ok. So the method actually takes an image and scales it down, and converts it to an nsdata. 
Try to come up with a name that lets us know what is going to happen?